### DISCUSSION TITLE
blog/Apple-Privacy

### DISCUSSION THREAD

**Discussion by giscus** [pycoder2000/blog#11] [General]
# blog/Apple-Privacy

Apple announced that it would begin checking content uploaded to iCloud Photos, against a list of known CSAM material. This is fine on it's own, however the system could easily be abused.

https://musing.vercel.app/blog/Apple-Privacy#comments-container

**Comment by pycoder2000** [comment]
Please leave your comments if you have any suggestions! 😁

**Comment by FetuTechTamer** [comment]
huhu is this even working?
